.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: go
+
+go:
+  - 1.9.2
+
+env:
+  global:
+   - ARCH=$(go env GOARCH)
+   - TAG=$(git describe --tags --always)
+
+sudo: required
+
+services:
+  - docker
+
+install:
+  - make bootstrap
+
+script:
+  - make && make test
+
+after_success:
+  - if [ "$TRAVIS_BRANCH" = "master" -a "$TRAVIS_PULL_REQUEST" = "false" ]; then
+      export REPO=openebs/node-disk-manager-$ARCH:$TAG;
```

Comment:
regular build is also using same, can push two images here =>
1. openebs/node-disk-manager:${VERSION}
2. openebs/node-disk-manager:latest

version we can get using "git rev-parse --short HEAD"

Reply:
Sure @pawanpraka1  
I will update it for pushing two images.